homeassistant/components/group/config_flow.py

```diff
@@ -67,6 +78,32 @@ async def binary_sensor_options_schema(handler: SchemaCommonFlowHandler) -> vol.
     }
 )
 
+SENSOR_OPTIONS = {
+    vol.Required(CONF_ALL, default=False): selector.BooleanSelector(),
```

Comment:
This is a very confusing property compared to what "all" means in other groups. In other groups, all affect toggle behavior, while here it is used for the availability of the included members.

IMHO, this should be renamed.